test/resources/FileLinks.spec.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+var stripe = require('../testUtils').getSpyableStripe();
+var expect = require('chai').expect;
+
+describe('FileLinks Resource', function() {
+  describe('retrieve', function() {
+    it('Sends the correct request', function() {
+      stripe.fileLinks.retrieve('fl_123');
+      expect(stripe.LAST_REQUEST).to.deep.equal({
+        method: 'GET',
+        url: '/v1/file_links/fl_123',
+        headers: {},
+        data: {},
+      });
+    });
+  });
+
+  describe('create', function() {
+    it('Sends the correct request', function() {
+      stripe.fileLinks.create({file: 'file_123'});
+      expect(stripe.LAST_REQUEST).to.deep.equal({
+        method: 'POST',
+        url: '/v1/file_links',
+        headers: {},
+        data: {file: 'file_123'},
+      });
+    });
+  });
+
+  describe('update', function() {
+    it('Sends the correct request', function() {
+      stripe.fileLinks.update('fl_123', {metadata: {key: 'value'}});
+      expect(stripe.LAST_REQUEST).to.deep.equal({
+        method: 'POST',
+        url: '/v1/file_links/fl_123',
```

Comment:
Quite minor and I didn't notice on your PHP pull, but file link tokens have a `test` or `live` in them. This is unlike most resources, but is needed because you're not using a key that would hint which mode to load it in.

It might be slightly more accurate if we used an identifier like `fl_live_123`.

Reply:
Wait, that's only for the `url` attribute though. File links have a regular token, but it starts with `link_`, not with `fl_`.

Reply:
Oh oops. Yeah you're right — I got confused seeing the `fl_`.

Reply:
totally misread that from an earlier link I saw that had fl in it. Thanks for catching, I'll also fix in PHP